Activities/OpenAISession/README.md

```diff
@@ -0,0 +1,7 @@
+## Agenda
```

Comment:
Usually I will try to fill out the README a little more - you don't have to add too much, but just think of what another person would need to know if they were leading this session. Could be as simple as [this](https://github.com/hylandtechoutreach/hackathon/tree/main/Activities/UxSession), plus your agenda.

Reply:
Thanks! Done!